commands/loop.md

```
@@ -0,0 +1,100 @@
+$ARGUMENTS
+
+## 概要
+
+Claude Codeの `--print` と `--allowedTools` オプションを使って、特定のタスクに特化したエージェントを別ペインで起動します。
+
+## 実行手順
+
+### 1. 新しいペインを作成
+
+```bash
+zellij action new-pane --direction right --name "pair-programmer"
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)